File: PiDevJDBC/src/PIDEV/Entitiess/Restaurant.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package PIDEV.Entitiess;

/**
 *
 * @author Admis
 */
public class Restaurant {

    private String id;
    private String adresse;
    private String menu;
    private String prog;
    private int  nbreP;
    private int telephone;

    
    

    public String getId() {
        return id;
    }

    public String getAdresse() {
        return adresse;
    }

    public String getMenu() {
        return menu;
    }

    public void setId(String id) {
        this.id = id;
    }

    @Override
    public String toString() {
        return "Restaurant{" + "id=" + id + ", adresse=" + adresse + ", menu=" + menu + ",Programme="+prog +",Telephone= "+ telephone +", Nombre de places="+ nbreP+ '}';
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public void setMenu(String menu) {
        this.menu = menu;
    }

    public Restaurant() {
    }

    public Restaurant(String id, String adresse, String menu, String prog, int telephone, int nbreP) {
        this.id = id;
        this.adresse = adresse;
        this.menu = menu;
        this.prog = prog;
        this.nbreP = nbreP;
        this.telephone = telephone;
    }

   

    public Restaurant(String id) {
        this.id = id;
       
    }

    public void setProg(String prog) {
        this.prog = prog;
    }

    public void setNbreP(int nbreP) {
        this.nbreP = nbreP;
    }

    public void setTelephone(int telephone) {
        this.telephone = telephone;
    }

    public String getProg() {
        return prog;
    }

    public int getNbreP() {
        return nbreP;
    }

    public int getTelephone() {
        return telephone;
    }

}

File: Symfony/TNMazesKey/src/Entity/Restaurant.php
<?php

namespace App\Entity;

use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

/**
 * Restaurant
 *
 * @ORM\Table(name="restaurant", uniqueConstraints={@ORM\UniqueConstraint(name="nom_unique", columns={"nom"})})
 * @ORM\Entity
 */
class Restaurant
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string|null
     *
     * @ORM\Column(name="nom", type="string", length=60, nullable=true)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="Adresse", type="string", length=200, nullable=false)
     */
    private $adresse;

    /**
     * @var string
     *
     * @ORM\Column(name="Menu", type="text", length=250, nullable=false)
     */
    private $menu;

    /**
     * @var string
     *
     * @ORM\Column(name="Prog", type="string", length=100, nullable=false)
     */
    private $prog;

    /**
     * @var int
     *
     * @ORM\Column(name="Telephone", type="integer", nullable=false)
     */
    private $telephone;

    /**
     * @var int
     *
     * @ORM\Column(name="NbreP", type="integer", nullable=false)
     */
    private $nbrep;

    /**
     * @var float|null
     *
     * @ORM\Column(name="note", type="float", precision=10, scale=0, nullable=true)
     */
    private $note;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(?string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getMenu(): ?string
    {
        return $this->menu;
    }

    public function setMenu(string $menu): self
    {
        $this->menu = $menu;

        return $this;
    }

    public function getProg(): ?string
    {
        return $this->prog;
    }

    public function setProg(string $prog): self
    {
        $this->prog = $prog;

        return $this;
    }

    public function getTelephone(): ?int
    {
        return $this->telephone;
    }

    public function setTelephone(int $telephone): self
    {
        $this->telephone = $telephone;

        return $this;
    }

    public function getNbrep(): ?int
    {
        return $this->nbrep;
    }

    public function setNbrep(int $nbrep): self
    {
        $this->nbrep = $nbrep;

        return $this;
    }

    public function getNote(): ?float
    {
        return $this->note;
    }

    public function setNote(?float $note): self
    {
        $this->note = $note;

        return $this;
    }
}
